docs/docs/getting-started/installation/aws.md

```diff
@@ -71,6 +115,25 @@ kubectl --kubeconfig ~/.airy/kube.conf get --namespace kube-system service traef
 Now that you have a running installation of Airy core on AWS you can connect it to messaging sources. Check out the
 [source documentation](/sources/introduction.md) to learn more.
 
+You can also check out the following content
+
+<ButtonBoxList>
+<ButtonBox
+    icon={<DiamondSVG />}
+    iconInvertible={true}
+    title='Sources'
+    description='Sources'
+    link='sources/introduction'
+/>
+<ButtonBox
+    icon={<DiamondSVG />}
+    iconInvertible={true}
+    title='To the Quick Start'
+    description='Learn the Airy Basics with our Quick Start'
+    link='getting-started/quickstart'
+/>
+</ButtonBoxList>
```

Comment:
I'd probably invert here? We do this thing of having the quickstart as last in a lot of places... and it's not very good I think. 

I'm not sure I'd have the source button either. It fells all a little random (/cc @sgarroum  as I think this was done according to specs)